src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/service/AccessRightService.java

```diff
@@ -78,147 +79,23 @@
     private final ServiceService serviceService;
     private final IdentifierService identifierService;
     private final GlobalConfService globalConfService;
-    private final EndpointRepository endpointRepository;
     private final EndpointService endpointService;
+    private final LocalGroupService localGroupService;
 
     @Autowired
     public AccessRightService(LocalGroupRepository localGroupRepository, GlobalConfFacade globalConfFacade,
             ClientRepository clientRepository, ServiceService serviceService, IdentifierService identifierService,
-            GlobalConfService globalConfService, EndpointRepository endpointRepository,
-            EndpointService endpointService) {
+            GlobalConfService globalConfService,
```

Comment:
private final LocalGroupRepository localGroupRepository is unused

Reply:
Removed